packages/components/src/dropdown/index.js

```diff
@@ -39,17 +36,6 @@ class Dropdown extends Component {
 		}
 	}
 
-	/**
-	 * When contents change height due to user interaction,
-	 * `refresh` can be called to re-render Popover with correct
-	 * attributes which allow scroll, if need be.
-	 */
-	refresh() {
```

Comment:
I wonder if we should deprecate it first before removing it :)

Reply:
Indeed ;)